tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -137,7 +144,16 @@ private void CompareDependencies(string path, IEnumerable<Tuple<string, int>> ex
                 .OrderBy(o => o.Item1, StringComparer.Ordinal)
                 .ToList();
 
-            Assert.Equal(expectedOrdered, foundDocIds);
+            Assert.Equal(expectedOrdered.Count, foundDocIds.Count);
+
+            foreach (var combined in expectedOrdered.Zip(foundDocIds, Tuple.Create))
+            {
+                var expectedItem = combined.Item1;
+                var actualItem = combined.Item2;
+
+                Assert.Equal(expectedItem.Item1, actualItem.Item1);
+                Assert.Equal(expectedItem.Item2, actualItem.Item2);
+            }
```

Comment:
I'd have hoped that Assert.Equal would have done the same thing as this code already. I take it that isn't the case.